test/doc/algo/reduce.cpp

```diff
@@ -0,0 +1,23 @@
+#include <eve/module/core.hpp>
+#include <eve/algo.hpp>
+#include <iostream>
+#include <vector>
+#include "detail/print.hpp"
+
+int main()
+{
+  std::vector<float> v(10);
+  auto vv = eve::views::iota(1.0f, 10);
+  eve::algo::copy(vv, v);
+  auto r1 = eve::algo::reduce(v, std::make_pair(eve::add, eve::zero), 0.0f);
```

Comment:
I don't believe you need to make pair here - these are the default operations.

Reply:
auto r1 = eve::algo::reduce(v); does not compile...
feel free to simplify that in a further pr.

Reply:
auto r1 = eve::algo::reduce(v, 0.0f);